src/main/java/sirius/search/Query.java

```diff
@@ -1219,6 +1222,8 @@ protected long transformCount(SearchRequestBuilder builder) {
         if (Microtiming.isEnabled()) {
             w.submitMicroTiming("ES", "COUNT: " + toString(true));
         }
+        indexAccess.addSelectTiming(w.elapsedMillis());
```

Comment:
wouldnt it make more sense to do this in SearchRequestBuilder.execute().actionGet() ?